src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/global.yml

```diff
@@ -1244,9 +1274,70 @@ Resources:
             - master
       Targets:
         - Arn:
-            !Join [ '', [ 'arn:aws:codepipeline:', !Ref 'AWS::Region', ':', !Ref 'AWS::AccountId', ':', !Ref CodePipeline ] ]
+            !Join [
+              "",
+              [
+                "arn:aws:codepipeline:",
+                !Ref "AWS::Region",
+                ":",
+                !Ref "AWS::AccountId",
+                ":",
+                !Ref CodePipeline,
+              ],
+            ]
           RoleArn: !GetAtt PipelineCloudWatchEventRole.Arn
           Id: adf-codepipeline-trigger-pipeline
+  TerraformLockTable:
+    Type: "AWS::DynamoDB::Table"
```

Comment:
Only when Terraform is enabled, as discussed above.

Reply:
Actually, this one should be removed. As it should be defined in the `regional.yml` file instead.
At the moment, it would conflict, as the `global` and `regional` stacks will overlap in the `global` region.